pycon/settings.py

```diff
@@ -719,126 +709,6 @@ def CONFERENCE_SCHEDULE_ATTENDEES(schedule, forecast):
 )
 
 
-def CONFERENCE_VIDEO_COVER_EVENTS(conference):
```

Comment:
Are this and the following method no longer used in `conference/settings.py`? I know there's a getattr with a fallback that's basically a no-op, but I'm not entirely clear on if it's safe to remove.

Reply:
as far as I can tell they are not used in the app; worst case scenario (for example if they were used in some adhoc scripts that are not part of general epcon) we can fetch them from git history.

Reply:
Looks like they're used a in shell command (`conference.management.commands.video_cover.Command`) and a view (`conference.views.covers`) - it's not clear to me at all what are those views used for though; do you know if they're safe to be removed?

Not a blocker, just thought that if we're removing this, it would be nice to remove the dependencies, too.